packages/datagateway-table/src/state/reducers/dgtable.reducer.test.tsx

```diff
@@ -1,103 +1,26 @@
 import {
-  sortTable,
-  fetchInvestigationsRequest,
-  fetchInvestigationsSuccess,
-  fetchInvestigationsFailure,
-  filterTable,
-  fetchDatasetsRequest,
-  fetchDatasetsSuccess,
-  fetchDatasetsFailure,
-  fetchDatafilesRequest,
-  fetchDatafilesSuccess,
-  fetchDatafilesFailure,
-  fetchInvestigationDatasetsCountRequest,
-  fetchInvestigationDatasetsCountSuccess,
-  fetchInvestigationDatasetsCountFailure,
-  fetchDatasetDatafilesCountRequest,
-  fetchDatasetDatafilesCountSuccess,
-  fetchDatasetDatafilesCountFailure,
-  fetchInstrumentsRequest,
-  fetchInstrumentsSuccess,
-  fetchInstrumentsFailure,
-  fetchFacilityCyclesRequest,
-  fetchFacilityCyclesSuccess,
-  fetchFacilityCyclesFailure,
-  downloadDatafileRequest,
-  downloadDatafileSuccess,
-  downloadDatafileFailure,
-  downloadDatasetRequest,
-  downloadDatasetSuccess,
-  downloadDatasetFailure,
-  fetchInvestigationCountRequest,
-  fetchInvestigationCountSuccess,
-  fetchInvestigationCountFailure,
-  fetchDatasetCountRequest,
-  fetchDatasetCountSuccess,
-  fetchDatasetCountFailure,
-  fetchDatafileCountRequest,
-  fetchDatafileCountSuccess,
-  fetchDatafileCountFailure,
-  fetchInstrumentCountRequest,
-  fetchInstrumentCountSuccess,
-  fetchInstrumentCountFailure,
-  fetchFacilityCycleCountRequest,
-  fetchFacilityCycleCountSuccess,
-  fetchFacilityCycleCountFailure,
-  clearTable,
-  loadFacilityName,
   loadFeatureSwitches,
   configureStrings,
-  loadUrls,
   loadBreadcrumbSettings,
   settingsLoaded,
-  fetchInvestigationDetailsRequest,
-  fetchInvestigationDetailsSuccess,
-  fetchInvestigationDetailsFailure,
-  fetchDatasetDetailsRequest,
-  fetchDatasetDetailsSuccess,
-  fetchDatasetDetailsFailure,
-  fetchInstrumentDetailsRequest,
-  fetchInstrumentDetailsSuccess,
-  fetchInstrumentDetailsFailure,
-  fetchDatafileDetailsRequest,
-  fetchDatafileDetailsSuccess,
-  fetchDatafileDetailsFailure,
-  fetchDownloadCartRequest,
-  fetchDownloadCartSuccess,
-  fetchDownloadCartFailure,
-  addToCartRequest,
-  addToCartSuccess,
-  addToCartFailure,
-  removeFromCartRequest,
-  removeFromCartSuccess,
-  removeFromCartFailure,
-  fetchAllIdsRequest,
-  fetchAllIdsSuccess,
-  fetchAllIdsFailure,
-  fetchInvestigationSizeRequest,
-  fetchInvestigationSizeSuccess,
-  fetchInvestigationSizeFailure,
-  fetchDatasetSizeSuccess,
 } from '../actions';
-import {
-  Investigation,
-  Dataset,
-  Datafile,
-  Instrument,
-  FacilityCycle,
-  DownloadCart,
-} from 'datagateway-common';
+import { dGCommonInitialState } from 'datagateway-common';
 
 describe('dgtable reducer', () => {
-  let state: DGTableState;
-  const invalidTimestamp = 0;
-  let validTimestamp = 0;
+  let state: StateType;
 
   beforeEach(() => {
-    state = { ...initialState };
-    validTimestamp = Date.now() + 1000;
+    // state = { ...dgTableInitialState };
+    state = JSON.parse(
+      JSON.stringify({
+        dgcommon: dGCommonInitialState,
+        dgtable: dgTableInitialState,
+      })
+    );
```

Comment:
Since you're only testing `DGTableReducer` - you don't actually need to set up the `dgcommon` state - you can instead just go back to what the original code had with `state = {...initialState}`